src/Hazelcast.Net.Tests/Core/DistributedEventSchedulerTests.cs

```diff
@@ -94,12 +101,12 @@ public async Task SchedulerTest()
                     args.Handled = true;
                     message += " (handling)";
                 }
-                logger.LogWarning(message);
+                HConsole.WriteLine(this, "WARN: " + message);
```

Comment:
Do we have plans to fully switch to `HConsole` instead of `ILogger` or is it only used for some internal logging?

Reply:
Everything `ILogger` is always compiled, and is public, we expect it to be useful for users. Everything `HConsole` is not compiled in our releases (it's marked with `[Conditional]` attributes) as it can be super super verbose. I am not 100% sure it's a "good thing" but it is quite useful.